azuracast/liquidsoap_custom.liq

```diff
@@ -193,6 +193,8 @@ playlist_keygen = audio_to_stereo(id="stereo_playlist_keygen", playlist_keygen)
 playlist_exit = playlist(id="playlist_exit",mode="randomize",reload_mode="watch",conservative=true,default_duration=10.,length=20.,"/var/azuracast/stations/lahmacun_radio/playlists/playlist_exit.m3u")
 playlist_exit = audio_to_stereo(id="stereo_playlist_exit", playlist_exit)
 
+playlist_graveyard_slot = playlist(id="playlist_graveyard_slot",mode="randomize",reload_mode="watch",conservative=true,default_duration=10.,length=20.,"/var/azuracast/stations/lahmacun_radio/playlists/playlist_graveyard_slot.m3u")
+playlist_graveyard_slot = audio_to_stereo(id="stereo_playlist_graveyard_slot", playlist_graveyard_slot)graveyard_slot
```

Comment:
I see a strange `graveyard_slot` string at the and of the line, which seems to be a typo.